service/RTC/MediaType.js

```diff
@@ -1,15 +1,9 @@
 /**
- * Enumeration of RTC media stream types
- * @type {{AUDIO: string, VIDEO: string}}
+ * The audio type.
  */
-var MediaType = {
-    /**
-     * The audio type.
-     */
-    AUDIO: "audio",
-    /**
-     * The video type.
-     */
-    VIDEO: "video"
-};
-module.exports = MediaType;
\ No newline at end of file
+export const AUDIO = "audio";
```

Comment:
is it better to do it this way then have an actual type that encompasses the two values?  maybe we want to at least name this like "MEDIA_TYPE_AUDIO" or something?  I actually think MediaType.AUDIO is nicer, but i dunno what the idiomatic way is.